src/util/Validate.js

```diff
@@ -0,0 +1,50 @@
+const {
+  BRIDGE_SIZE,
+  MOVING_SPACE,
+  ERROR_MESSAGE,
+  RETRY_OR_QUIT,
+} = require("./Constant");
+
+const isNotNumber = (size) => {
+  const exceptNumber = /[^0-9]/;
+  if (exceptNumber.test(size)) return true;
+  return false;
```

Comment:
```suggestion
  return exceptNumber.test(size);
```
이렇게 줄여 쓸 수 있을 것 같습니다.